src/index.js

```diff
@@ -118,7 +79,11 @@ ipcRenderer.on('export', (event, message) => {
 
 ipcRenderer.on('export-tags', (event, exportConfig) => {
   addLoader();
-  detection.export(videotagging.imagelist, exportConfig.exportFormat, exportConfig.exportUntil, exportConfig.exportPath, testSetSize, () => {
+  let imagePaths;
+  if(videotagging.imagelist){
+    imagePaths = videotagging.imagelist.map((filepath) => {return path.join(videotagging.sourceDir,filepath)})
```

Comment:
You don't need the return statement or the wrapping parens.

ex) `imagePaths = videotagging.imagelist.map((filepath) => path.join(videotagging.sourceDir,filepath))`